### ISSUE TITLE
Cannot use None in NewType

### ISSUE THREAD

**Issue by RocketRace** [microsoft/pyright#4019]
**Describe the bug**
`None` passed as the second argument to `NewType` raises an error diagnostic.

**To Reproduce**
```python
from typing import NewType

Foo = NewType("Foo", None) # error: Expected class as second argument to NewType
reveal_type(Foo) # information: Type of "Foo" is "Unknown"
```

**Expected behavior**
`NewType("Foo", None)` should succeed, and allow `Foo(None)` to typecheck.

**VS Code extension or command-line**
* pyright 1.1.273
* pylance v2022.10.10

**Comment by erictraut** [comment]
Pyright's behavior is correct here. The second argument to `NewType` must be a subclassable class. The same error is emitted by mypy. `None` is an object instance, not a class that can be subclassed.
